src/test/resources/features/Roles.feature

```diff
@@ -2,33 +2,23 @@ Feature: Role features
 
 
   Background:
-    Given a database for users and roles
-    And an authorized client
+    Given a Database for users and roles
+    And an AuthorizedClient that is authorized through Feide
 
+    And an ExistingRole with role-name "ExistingRole" that exists in the Database
+    And a NonExistingRole with role-name "NonExistingRole" that does not exist in the Database
+    And a NewRole with role-name "NewRole" that does not yet exist in the Database
+
+#
   Scenario: Authorized client creates Role
-    Given the authorized client forms a "POST" request
-    And the request contains a JSON body with following key-value pairs
-      | key      | value   |
-      | rolename | theRole |
-      | type     | Role    |
-    When the authorized client sends the request to add a new Role
-    Then a new role is stored in the database
-    And the description of the role is returned to the authorized client
+    When the AuthorizedClient requests to add the NewRole to the Database
+    Then an OK message is returned
+    And a role description is returned
```

Comment:
If you want to describe the role description you can do that in one of the scenarios